### DISCUSSION TITLE
Is it possible to run the cloudnative-pg without operator and as a single instance?

### DISCUSSION THREAD

**Discussion by NiklasMayerhauserGH** [cloudnative-pg/cloudnative-pg#6082] [Q&A]
Reason for that would be less maintaining, if I need also a single running postgres without replication.

**Comment by alexbarcelo** [comment]
If you want for "magic to happen" when a CRD manifest is put inside kubernetes, then you need an operator (by definition).

If you want a Postgres cluster without extra steps and no CRD, you can deploy a Postgres cluster without extra steps (e.g. with the Bitnami chart).

I believe I missed your point. Could you elaborate?
